app/views/wikis/edit.html.erb

```diff
@@ -15,12 +15,14 @@
             <%= f.text_area :body, autofocus: true, class: 'form-control fr-view', id: 'wysiwyg-editor' %>
           </div>
         </div>
-        <div class="form-group">
-          <%= f.label :private, class: 'col-lg-2 control-label' %>
-          <div class="col-lg-10">
-            <%= f.check_box :private, class: 'form-control' %>
+        <% unless @user.standard? %>
```

Comment:
It would be very cool if a Policy could handle this, instead of directly querying the user's role. I can see that the 'ability' here, i.e. ability to create a private wiki, doesn't fit so cleanly into the default policy scheme which follows the HTTP rest actions.